### PR TITLE
[IMP] l10n_ec: Improvements EC

### PR DESCRIPTION
**Pull request by stevTresCloud**
- Add Customer accounts receivable for point-of-sale module
- Add new accounting account group for customer checks
- Add the following accounting accounts
    - 1102050103, Customer accounts receivable for point of sale
    - 1102050201, Customer checks to be deposited
    - 1102050202, Protested checks 
    - 11020705, Employee Loans
    - 11020706, Advance for overdraft in Role
    - 11050108, VAT paid on withholding tax on credit card profits
    - 11050205,Bank Withholdings to be Reconciled Transitory Account
    - 110505, Credit Card Withholding Tax Paid at Source
    - 110506,Taxable Base of Withholding Taxes on Sales
    - 5202040503,IESS contribution of personnel assumed by the company
    - 52011805, Credit card commissions

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_ec/data/res_country_data.xml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <record id="base.ec" model="res.country">
+            <field name="zip_required">False</field>
```
Comment (vbe-odoo) [inline addons/l10n_ec/data/res_country_data.xml:5]: @stevTresCloud we already have this implemented, zip code is not required in eCommerce, so we don't need this change in this PR.
**Here is where you can check on this:**
1. [PR that introduced eCommerce made zip not required](https://github.com/odoo/odoo/pull/142730/files#diff-b74c3d7f410ba41cdaaabae451d8fefb79fd964ea9cf476b93e96696c8215206)
2. [File where you can see this in master](https://github.com/odoo/odoo/blob/master/odoo/addons/base/data/res_country_data.xml#L415)
3. Here is an screenshot of how it looks like when you installed `l10n_ec_website_sale`
image

cc' @jco-odoo

Comment (vbe-odoo) [reply addons/l10n_ec/data/res_country_data.xml:5]: All the rest looks good, if you want to take a look at it @jco-odoo

File: addons/l10n_ec/data/res_country_data.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <data>
+        <record id="base.ec" model="res.country">
+            <field name="address_format" eval="'%(street)s\n%(street2)s\n%(city)s\n%(country_name)s'" />
```
Comment (jco-odoo) [inline addons/l10n_ec/data/res_country_data.xml:5]: base module

File: addons/l10n_ec/models/res_partner.py [hunk 3]
```diff
@@ -30,7 +30,9 @@ def get_ats_code_for_partner(cls, partner, move_type):
         Returns ID code for move and partner based on subset of Table 2 of SRI's ATS specification
         """
         partner_id_type = partner._l10n_ec_get_identification_type()
-        if move_type.startswith('in_'):
+        if partner.vat and verify_final_consumer(partner.vat):
```
Comment (vbe-odoo) [inline addons/l10n_ec/models/res_partner.py:33]: @stevTresCloud @pepetreshere let's separate this into a new PR pointing to V17.0

**Comment by pepetreshere** [comment]
Let's wait for us to make a testing video of CoA update/migration issues

**Comment by vbe-odoo** [comment]
@jco-odoo ready for your final check and merge.
Thanks!

**Comment by jco-odoo** [comment]
@robodoo r+

File: addons/l10n_ec/data/template/account.group-ec.csv [hunk 4]
```diff
@@ -9,6 +9,7 @@
 "ec1102","1102","Financial assets","ec11","Activos financieros"
 "ec110205","110205","Unrelated notes and accounts receivable","ec1102","Documentos y cuentas por cobrar cliente no relacionados"
 "ec11020501","11020501","Non-interest bearing notes and accounts receivable from unrelated ordinary activities interest-bearing","ec110205","Documentos y cuentas por cobrar cliente no relacionados de actividades ordinarias que no generen intereses"
+"ec11020502","11020502","Customer checks","11020502","Cheques de clientes"
```
Comment (jco-odoo) [inline addons/l10n_ec/data/template/account.group-ec.csv:12]: @stevTresCloud Something is off with the parent_id here. I guess that is why the staging failed.

Comment (rrvc12) [reply addons/l10n_ec/data/template/account.group-ec.csv]: @jco-odoo Hi thanks!. I fixed it
You can try again please.

**Comment by jco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**